### PR TITLE
[MIRROR]  Admin lua scripting (#65635)

### PR DESCRIPTION
**Pull request by Zonespace27**
Co-authored-by: Mothblocks <35135081+Mothblocks@users.noreply.github.com>

## About The Pull Request
Mirrors tgstation/tgstation#65635 and tgstation/tgstation#68649
Yes i committed to master shut your fuck

:cl:
code: Added auxlua
/:cl:

### CODE REVIEW

File: code/controllers/configuration/entries/lua.dm
Comment (GoldenAlpharex) [inline code/controllers/configuration/entries/lua.dm:2]: Do we need to add a config entry for that? If so, where?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Just make sure to tell us Keyholders in maintainer-talk what needs to be ran once this PR gets merged so that we can have this without it causing runtimes out of the wazoo.

File: config/lua.txt
Comment (Zonespace27) [inline config/lua.txt:1]: this config file @GoldenAlpharex

File: code/controllers/configuration/entries/lua.dm
Comment (Zonespace27) [reply code/controllers/configuration/entries/lua.dm:2]: Read other comment, add it where other config files go if it needs to be added

**Pull request merged**
